reference.md

```diff
@@ -704,3 +707,73 @@ Field Name | Presence | Type | Description
   }
 }
 ```
+
+### booking_rules.json
+
+This file defines rules about how to book a ride in advance and is to be used instead `wait_times` if users can not order a ride in real-time. With ride bookings, no wait times are applied. If available, users can either book a ride for the same day with an advance notice or for a futur day. 
+
+The following fields are all attributes within the main "data" object for this feed.
+
+
+| Field Name | Presence | Type |Description |
+ ---------- | ---- | -------- | ----------- |
+ `booking_rules` |  REQUIRED  | Array | Array that contains one object per booking rules as defined below. |
+\-&nbsp;`from_s2_cells` | Conditionally REQUIRED | Array | The reference to one or many S2CellID that cover the area of the wait time update. Information on S2 cells can be found here https://s2geometry.io/. Required if `from_zone_ids` field is not populated. Forbidden otherwise.
+\-&nbsp;`to_s2_cells` | OPTIONAL | Array | The reference to one or many S2CellID that cover the area of the destination. Information on S2 cells can be found here https://s2geometry.io/. Optional if `from_s2_cells` field is populated. Forbidden otherwise.
+\-&nbsp;`from_zone_ids` | Conditionally REQUIRED | Array | One or many ID from a zone defined in `zones.json`  that cover the area of the wait time update. Required if `from_s2_cells` field is not populated. Forbidden otherwise.
+\-&nbsp;`to_zone_ids` | OPTIONAL | Array | One or many ID from a zone defined in `zones.json`  that cover the area of the destination. Optional if `from_zone_ids` field is populated. Forbidden otherwise.
+ \-&nbsp; `booking_type` | REQUIRED | Enum | Indicates how far in advance booking can be made. <br><br>Valid options are:<br>`0` - Up to same-day booking with advance notice.<br>`1` - Up to prior day(s) booking. |
+ \-&nbsp; `prior_notice_duration_min` | Conditionally REQUIRED | Integer | Minimum number of minutes before travel to make the request. REQUIRED for `booking_type=0`. Forbidden otherwise. |
```

Comment:
```suggestion
 \-&nbsp; `prior_notice_duration_min` | Conditionally REQUIRED | Integer | Minimum number of minutes before travel to make the request. REQUIRED for `booking_type=0`. FORBIDDEN otherwise. |
```

Reply:
I've spotted a few other lower case `Forbidden`, I'll replace them too.